server/Makefile

```diff
@@ -20,17 +20,16 @@ binbase = \
 	pbench-base.py\
 	pbench-base.sh
 
-# pbench-server-activate-create-crontab is needed by the ansible installation playbook.
-# pbench-remote*, pbench-satelliter-state-change.py and pbench-server-set-result-state
-# are needed on satellites, but we install them everywhere.
+# pbench-server-activate-create-crontab is needed by the ansible installation
+# playbook.  pbench-remote* and pbench-satelliter-state-change.py are needed on
```

Comment:
I almost hate to point this out, but you could have fixed the "satelliter" misspelling while you were here. Since it's not part of your change, I'll pre-resolve this. 😆

Reply:
Mispelling?  I don't see a mispelling...